torchaudio/functional/functional.py

```diff
@@ -1444,3 +1446,82 @@ def resample(
                                               resampling_method, beta, waveform.device, waveform.dtype)
     resampled = _apply_sinc_resample_kernel(waveform, orig_freq, new_freq, gcd, kernel, width)
     return resampled
+
+
+def character_edit_distance(r: str, h: str) -> int:
+    """
+    Calculate the character level edit (Levenshtein) distance between two strings.
+
+    The function computes an edit distance allowing deletion, insertion and substitution.
+    The result is an integer. Users may want to normalize by the length of the reference.
```

Comment:
Can you also mention if this distance is symmetric or asymmetric?

Reply:
If the distance is symmetric, I think function arguments should not be called hypothesis and reference. That is the context of caller side and it does not concern in this function.

Reply:
Yes, it is symmetric, I'll change the name too string1 and string2 for character_edit_distance and sentence1 and sentence2 for word_edit_distance.